onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -313,6 +313,7 @@ def _where_input_wrangler(
     "nn.functional.adaptive_avg_pool1d": nn_ops.aten_adaptive_avg_pool1d,
     "nn.functional.adaptive_avg_pool2d": nn_ops.aten_adaptive_avg_pool2d,
     "nn.functional.adaptive_avg_pool3d": nn_ops.aten_adaptive_avg_pool3d,
+    # "nn.functional.cross_entropy_loss": nn_ops.aten_cross_entropy_loss,  # not is OPS_DB
```

Comment:
Can we create an OpInfo for it? I feel that this is an op we need to test.

Reply:
Jay has an example https://github.com/microsoft/onnx-script/blob/818d75c9e142f4fa9d082a2af75f05726e41c6dd/onnxscript/tests/function_libs/torch_aten/extra_opinfo.py

Reply:
https://github.com/pytorch/pytorch/blob/da41003b5f2f1709c1e1e524165c000e4f6e413e/torch/testing/_internal/common_methods_invocations.py#L11520

Is this relavant or reusable?

Reply:
yes I'm using the 'cross_entropy' as test case to test cross_entropy_loss